politeiawww/api/pi/v1/v1.go

```diff
@@ -72,3 +155,67 @@ type VoteMetadata struct {
 	// in the runoff vote.
 	LinkTo string `json:"linkto,omitempty"`
 }
+
+var (
+	// BillingStatuses contains the human readable billing statuses.
+	BillingStatuses = map[BillingStatusT]string{
+		BillingStatusInvalid:   "invalid",
+		BillingStatusClosed:    "closed",
+		BillingStatusCompleted: "completed",
+	}
+)
+
+// BillingStatusT represents the billing status of a proposal that has been
+// approved by the Decrd stakeholders.
+type BillingStatusT uint32
+
+const (
+	// BillingStatusInvalid is an invalid billing status.
+	BillingStatusInvalid BillingStatusT = 0
+
+	// BillingStatusClosed represents a proposal that was approved by
+	// the Decred stakeholders, but has been closed by an admin prior
+	// to the proposal being completed. The most common reason for this
+	// is because a proposal author failed to deliver on the work that
+	// was funded in the proposal. A closed proposal can no longer be
+	// billed against.
+	BillingStatusClosed BillingStatusT = 1
+
+	// BillingStatusCompleted represents a proposal that was approved
+	// by the Decred stakeholders and has been successfully completed.
+	// A completed proposal can no longer be billed against. A proposal
+	// is marked as completed by an admin.
+	BillingStatusCompleted BillingStatusT = 2
+)
+
+// BillingStatusChange represents the structure that is saved to disk when
+// a proposal has its billing status updated. Some billing status changes
+// require a reason to be given.
+//
+// Signature is the admin signature of the Token+Status+Reason.
+type BillingStatusChange struct {
```

Comment:
Make the same changes to the politeiawww API that are being made to the plugin API.